python/pyspark/testing/connectutils.py

```diff
@@ -58,7 +63,23 @@ class PlanOnlyTestFixture(unittest.TestCase):
 
     connect: "MockRemoteSession"
 
+    @classmethod
+    def _read_table(cls, table_name: str) -> "DataFrame":
+        return DataFrame.withPlan(Read(table_name), cls.connect)  # type: ignore
+
+    @classmethod
+    def _udf_mock(cls, *args, **kwargs):
+        return "internal_name"
+
     @classmethod
     def setUpClass(cls: Any) -> None:
         cls.connect = MockRemoteSession()
         cls.tbl_name = f"tbl{uuid.uuid4()}".replace("-", "")
```

Comment:
We shouldn't actually use uuid here but call `DROP IF EXISTS` before and after running the tests. Since this PR doesn't target to address these, I am fine with doing separately too.

Reply:
ah yes.

I will follow up. Python side testing framework needs more refinement for sure.